### PR TITLE
CORS Support doc and config

### PR DESCRIPTION
**Pull request by elosoman**
After splitting the apolloServer and apolloClient based on the meteor-starter kit repo I ran into CORS errors which prevented the client from accessing the apollo-server. The remedy was to configure the express engine to use the npm cors. The docs have been updated to reflect proper setup.

### CODE REVIEW

File: source/apollo-client/index.md [hunk 1]
```diff
@@ -64,33 +64,9 @@ const client = new ApolloClient({
 });
 ```
 
-To automatically send cookies for the current domain (same server-client environment), the credentials option must be provided,
+<h4 id="corsSupport" title="cors support">CORS support</h3>
```
Comment (abhiaiyer91) [inline source/apollo-client/index.md]: we should link to the section that we want people to go to right?

File: source/apollo-client/index.md [hunk 2]
```diff
@@ -64,33 +64,9 @@ const client = new ApolloClient({
...
 
-```js
```
Comment (abhiaiyer91) [inline source/apollo-client/index.md]: why was this removed? Cause it just didnt work lol?

Comment (elosoman) [inline source/apollo-client/index.md]: That was my observation. It appears to be a server issue resolved by adding cors npm. I may have overstepped with this removal and would be glad to retract.

Comment (abhiaiyer91) [inline source/apollo-client/index.md]: @helfer what do you think?
